core/web/resolver/helpers.go

```diff
@@ -3,14 +3,14 @@ package resolver
 import (
 	"database/sql"
 	"fmt"
-	"strconv"
 	"strings"
 
 	"github.com/graph-gophers/graphql-go"
 	"github.com/pkg/errors"
 
 	"github.com/smartcontractkit/chainlink/core/assets"
 	"github.com/smartcontractkit/chainlink/core/bridges"
+	stringutils "github.com/smartcontractkit/chainlink/core/utils/string_utils"
```

Comment:
You can get rid of the name declaration here. It will automatically be `stringutils` from  the package name